src/components/ui/ImageGallery/ImageGallerySelector.tsx

```diff
@@ -1,121 +1,84 @@
-        icon={<BackwardArrowIcon color="#323845" />}
-        onClick={() => {
-          if (sliderState.sliding) {
-            return
-          }
-
-          slide('previous', sliderDispatch)
-        }}
-      />
-      <IconButton
-        aria-label="forward slide image selector"
-        icon={<ForwardArrowIcon color="#323845" />}
-        onClick={() => {
-          if (sliderState.sliding) {
-            return
-          }
-
-          slide('next', sliderDispatch)
-        }}
-      />
-
-      <div {...otherProps} {...handlers}>
-        <div
-          data-carousel-track
-          style={{
-            display: 'flex',
-            transition: sliderState.sliding ? `transform 400ms` : undefined,
-            width: `${(elementCount * 100) / itemsPerPage}%`,
-            transform: `translate3d(${getTransformValue(
-              elementCount,
-              sliderState.currentPage,
-              itemsPerPage
-            )}%, 0, 0)`,
-          }}
-          onTransitionEnd={() => {
-            sliderDispatch({
-              type: 'STOP_SLIDE',
-            })
+  return false
+}
 
-            if (sliderState.currentItem >= elementCount) {
-              sliderDispatch({
-                type: 'GO_TO_PAGE',
-                payload: {
-                  pageIndex: 0,
-                  shouldSlide: false,
-                },
-              })
-            }
+function ImageGallerySelector({ images, onSelect, currentImageIdx }: Props) {
+  const elementsRef = useRef<HTMLDivElement>(null)
 
-            if (sliderState.currentItem < 0) {
-              sliderDispatch({
-                type: 'GO_TO_PAGE',
-                payload: {
-                  pageIndex: sliderState.totalPages - 1,
-                  shouldSlide: false,
-                },
-              })
-            }
-          }}
-        >
-          {elements.map((el, idx) => {
-            return (
-              <div key={idx} style={{ width: `${100 / elementCount}%` }}>
-                <div className="flex justify-center items-center w-full">
-                  {el}
-                </div>
-              </div>
-            )
-          })}
-        </div>
+  return (
+    <section
+      data-fs-image-gallery-selector
+      aria-roledescription="carousel"
+      aria-label="Product images"
+    >
+      {hasScroll(elementsRef.current) && (
+        <IconButton
+          aria-label="backward slide image selector"
+          icon={<Icon name="ArrowLeft" width={24} height={24} />}
+          onClick={() => moveScroll(elementsRef.current, -200)}
+        />
+      )}
+      <div data-fs-image-gallery-selector-elements ref={elementsRef}>
+        {images.map((image, idx) => {
+          return (
+            <Button
+              key={idx}
+              data-thumbnail-button={
+                idx === currentImageIdx ? 'selected' : 'true'
+              }
+              aria-label={`Load ${image.alternateName} - Image ${idx + 1} of ${
```

Comment:
Do we really need to mention "load"?
```suggestion
              aria-label={`${image.alternateName} - Image ${idx + 1} of ${
```